modules/ROOT/pages/index.adoc

```diff
@@ -8,26 +15,96 @@
 
 :console: 0
 
-This is the Neo4j Operations Manual, which includes all the operational details and instructions for installing and deploying Neo4j on-premise and in the cloud.
+[.display.cards.selectable]
 
-For all information on *upgrading and migrating Neo4j*, see the link:{neo4j-docs-base-uri}/upgrade-migration-guide/current/[Neo4j Upgrade and Migration Guide].
+== Manage and monitor your deployments
 
-For more information on link:https://neo4j.com/aura/[Aura], the Neo4j fully managed cloud service, see the link:{neo4j-docs-base-uri}/aura[Neo4j Aura Manual].
+Find here details and instructions for installing and deploying Neo4j on-premises and in the cloud.
+Follow tutorials and find administrative commands to help you manage and monitor your projects.
+The latest version of Neo4j is *{neo4j-version-exact}*.
```

Comment:
```suggestion
The Operations manual includes all the operational details and instructions for installing, deploying, and operating Neo4j in a self-managed environment. +
For all information on upgrading and migrating Neo4j, see the [Neo4j Upgrade and Migration Guide](https://neo4j.com/docs/upgrade-migration-guide/current/). +
For more information on [Aura](https://neo4j.com/aura/), the Neo4j fully managed cloud service, see the [Neo4j Aura Manual](https://neo4j.com/docs/aura). 

The latest version of Neo4j is *{neo4j-version-exact}*.
```

Reply:
I made another suggestion, please check the new version